src/main.cpp

```diff
@@ -59,8 +59,10 @@ int main(int argc, char** argv)
     daemon->run();
 
     if ((argc == 2) && (strcmp(argv[1], "--sync") == 0)) {
-        qDebug() << "Start manual sync";
-        QTimer::singleShot(1000, daemon, SLOT(syncAll()));
+        // We need to wait a little bit so we realize that we're connected to
+        // the internet
+        qDebug() << "Starting manual sync in 10 seconds.";
```

Comment:
Isn't 10 seconds too much, though? In theory, the time needed to get the network status is the time of making a D-Bus call, so even one second should be enough.

Reply:
No, getting the network status can take quite a while. 1 second was not enough.